generators/server/templates/prepareMicrofrontends.sh

```diff
@@ -18,10 +18,22 @@ function processWidget() {
     (
        cd "$WIDGET_PATH"
 
+       THIS=$BASHPID
        echo "Installing micro-frontend dependencies"
-       npm install
+       npm install || return $?
```

Comment:
@Kerruba explicit control of install result

the parent script runs a "set -e" but that setting is not respected in subshells